cengage/react-magma
# fix(NativeSelect): Add `aria-describedby` to helper and error messages to associate them with the `Select`.

Issue: #1251

## What I did

Add `aria-describedby` to helper and error messages to associate them with the `Select`.

## Screenshots:


## Checklist 

## How to test

Open storybook or docs -> Open `NativeSelect` with `error message` or `helper message` -> Open DevTools -> Check that message has the correct `aria-describedby`.
